pkgs/development/interpreters/bats/default.nix

```
@@ -22,6 +22,9 @@ stdenv.mkDerivation rec {
 
   inherit doCheck;
   checkPhase = ''
+    # TODO: reenable this test when fixed
```

```suggestion
    # TODO: cut if https://github.com/bats-core/bats-core/issues/418 allows
```
I picked at this a little bit to figure out what was going on and opened an issue. I'm not certain it'll result in us being able to drop the skip, but at least we'll have an external reference to monitor for whether this has become actionable :)

---

Thank you! :pray: